AgentDiscoveries-Frontend/app/src/components/admin/entity.jsx

```diff
@@ -23,7 +23,10 @@ export default class Entity extends React.Component {
 
     getEntityRow() {
         return Object.keys(this.props.entity).map(key =>
-            <td key={key}>{this.props.entity[key].toString()}</td>);
+        {
+            const display = this.props.entity[key] === null ? '' : this.props.entity[key].toString(); //This replaces null elements with an empty string.
```

Comment:
If you're adding a comment, I'd usually add it directly above the line it's commenting on, rather than after.
That way if the screen isn't wide enough to hold it, the developer will still be aware of it.

In this case, I'm not sure that the comment adds any clarity though - it's useful if you aren't familiar with the syntax, but I think the syntax is probably clear enough as is, and comments have a strong tendency to quickly get out of date and become invalid/confusing - I'd probably remove the comment